skaha/src/main/java/org/opencadc/skaha/session/PostAction.java

```diff
@@ -620,17 +623,24 @@ public void createSession(String type, String image, String name, Integer cores,
         jobLaunchString = setConfigValue(jobLaunchString, SOFTWARE_LIMITS_CORES, cores.toString());
         jobLaunchString = setConfigValue(jobLaunchString, SOFTWARE_LIMITS_RAM, ram + "Gi");
         jobLaunchString = setConfigValue(jobLaunchString, SOFTWARE_LIMITS_GPUS, getGPUResourceLimit(gpus));
+
+        try {
+            final int majorNVIDIACUDAVersion = CommandExecutioner.getMajorNvidiaCudaGPUVersion();
+            jobLaunchString = setConfigValue(jobLaunchString, SOFTWARE_GPU_NVIDIA_CUDA_MAJOR_VERSION, Integer.toString(majorNVIDIACUDAVersion));
```

Comment:
Users already have access to the GPU version through their software, but this may be useful, not sure.

The general idea is to allow users to land on the right GPU (brand, version, gpu-core count).  I think the ideas from ExecutionBroker are useful here and will help us align with that potential integration. 

We currently expose at /context the content of k8s-resources.  So this is just a static config reflecting the underlying capabilities of the cluster.  Ideally, those values should come from the cluster instead.  However, that is probably beyond the scope of this story.  Also beyond the scope is adding the 'brokering' part of client interaction.

So I think, for now at least, the story is to simply let users specify, through API params, those 3 gpu conditions.  I haven't gone through this whole PR yet but I'm guessing that a lot of that is already there.  Let's chat about it tomorrow.

Reply:
Thanks.  According to [CADC-13476](https://cadc-ccda.atlassian.net/browse/CADC-13476), we wanted the Major CUDA version supplied.  This way scripts can look it up.

Also, there should be two (2) parameters specified; the `gpu-type` and the `gpus` (count) parameter.